### ISSUE TITLE
Email: Create interface and logic for one and only one primary email address per contact

### ISSUE THREAD

**Issue by arborrow** [arborrow/montserrat#206]
Should probably do something similar for phone and addresses

**Comment by arborrow** [comment]
I went ahead and ran a few queries to ensure that every contact has one and only one primary email address. 

To find if there are contacts with more than one primary email address:
```
SELECT e.contact_id 
FROM email as e
LEFT JOIN contact as c ON (c.id=e.contact_id)
WHERE e.deleted_at IS NULL AND e.email IS NOT NULL
GROUP BY e.contact_id
HAVING sum(e.is_primary) > 1;

```
To find if there are contacts with an email address but none of them are listed as a primary email address:
```

SELECT e.contact_id 
FROM email as e
LEFT JOIN contact as c ON (c.id=e.contact_id)
WHERE e.deleted_at IS NULL AND e.email IS NOT NULL
GROUP BY e.contact_id
HAVING sum(e.is_primary) = 0
```

**Comment by arborrow** [comment]
fixed by #267
